packages/fast-form-generator-react/src/form/form.utilities.ts

```diff
@@ -218,6 +227,156 @@ export function getComponentTracker(
     return updatedComponentTracker;
 }
 
+/**
+ * Creates a schema location from a data location
+ */
+export function mapSchemaLocationFromDataLocation(dataLocation: string, data: any, schema: any): string {
+    const squareBracketRegex: RegExp = /\[(\d+?)\]/g;
+    let normalizedDataLocation: string = dataLocation.replace(squareBracketRegex, `.$1`); // convert all [ ] to . notation
```

Comment:
I want to standardize the input to make sure its always the correct format. For example, correct lodash notation could be a["b"].0.c, I want to make sure this is always a.b[0].c so that I can split using "." and reconstruct the schema location with these items.